native/packages/react-native-bpk-component-alert/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "react-native-bpk-component-alert",
+  "version": "0.0.01",
+  "main": "index.js",
+  "description": "Backpack React Native switch component.",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:Skyscanner/backpack.git"
+  },
+  "author": "Backpack Design System <backpack@skyscanner.net>",
+  "license": "Apache-2.0",
+  "peerDependencies": {
+    "react": "^16.0.0-alpha.1",
+    "react-native": ">= 0.47.0"
+  },
+  "dependencies": {
+    "bpk-tokens": "^27.0.0",
+    "prop-types": "^15.5.8",
+    "react-native-bpk-theming": "^1.0.53"
```

Comment:
I don't think `react-native-bpk-theming` is being used in this project, so it doesn't need to be listed here.